### PR TITLE
Implement tokenlist update script

### PR DESCRIPTION
**Pull request by bryanwb**

### CODE REVIEW

File: src/pricefeeds.json [hunk 1]
```diff
@@ -0,0 +1,5258 @@
...
+    "name": "TUSD / USD",
+    "assetName": "TUSD",
+    "address": "0xec746eCF986E2927Abd291a2A1716c940100f8Ba",
+    "path": "tusd-usd",
+    "input": "TUSD",
+    "output": "USD",
+    "type": "crypto",
+    "updateInterval": 600
+  },
+  {
```
Comment (yodlmeister) [inline src/pricefeeds.json:722]: check

File: src/pricefeeds.json [hunk 2]
```diff
@@ -0,0 +1,5258 @@
...
+    "input": "EOS",
+    "output": "USD",
+    "type": "crypto",
+    "updateInterval": 600
+  },
+  {
+    "decimals": 8,
+    "chainId": 137,
+    "name": "ETC / USD",
+    "assetName": "ETC",
```
Comment (yodlmeister) [inline src/pricefeeds.json:1470]: is that an actual token on that chain?

Comment (bryanwb) [reply src/pricefeeds.json:1470]: No, but there is a pricefeed
Screenshot from 2024-06-21 14-25-11

File: src/pricefeeds.json (hunk 1 above)
Comment (bryanwb) [reply src/pricefeeds.json:1403]: resolved

**Pull request merged**